src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml.cs

```diff
@@ -62,18 +63,9 @@ private enum States
         /// <summary>
         /// Avatar view model
         /// </summary>
-        public ByteArrayViewModel vmAvatar { get; private set; } = new ByteArrayViewModel();
-
-        /// <summary>
-        /// Current user display name
-        /// </summary>
-        public static string DisplayName => AzureDevOps.DisplayName;
-
-        /// <summary>
-        /// Current user email
-        /// </summary>
-        public static string Email => AzureDevOps.Email;
+        public ConfigurationViewModel vmConfig { get; private set; } = new ConfigurationViewModel();
 
+        #pragma warning restore CA1822
```

Comment:
>CA1822 [](start = 32, length = 6)

You're restoring CA1822 without suppressing it first. Seems like this line should go away?

Reply:
I see no use for it, I'll remove it for now, @jalkire you can check later if you think we need to keep it